editor/editor.tsx

```diff
@@ -776,6 +776,9 @@ function Editor(props: RouteComponentProps) {
                     </span>
                 ) : null}
                 <input type="hidden" id="spec-url-exporter" />
+                <a href="http://gosling-lang.org/" title="Go to the Gosling Project" className="mr-1">
```

Comment:
I think we can also open a new tab instead to prevent users from losing their code on the editor. 

```suggestion
                <a href="http://gosling-lang.org/" title="Go to the Gosling Project" className="mr-1" target="_blank">
```